GetStream/stream-chat-android
# Cleanup after migrating UI to `OfflinePlugin`

### 🎯 Goal

Our efforts to Migrate our UI modules over to OP require some minor cleanup effort.

### 🛠 Implementation details

- Remove unnecessary OptIns now that `OfflinePlugin` and the related structure is no longer marked with `@InternalStreamChatApi`
- Remove unnecessary message preloading while requesting `ChannelState` where there is no need for it.

### 🎨 ~~UI Changes~~

**No UI Changes**

### 🧪 Testing

Test basic functionalities in UI components such as 

- Checking that typing in a message produces a typing update on another device
- Send a message
- Check that `MessageListHeaderView` is behaving as expected (Online status, typing updates, the correct number of members, etc)

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![cleanup](https://user-images.githubusercontent.com/37080097/158628096-0713f99a-a503-40dc-84bf-92428f4bf853.gif)
